src/main/webapp/i18n/de/exam.json

```diff
@@ -4,7 +4,22 @@
             "title": "Exam"
```

Comment:
Missing "artemisApp.exam.created" and "artemisApp.exam.updated". As stated by @pzdr7

Reply:
Implemented in 5eb7fbc5365cc7ca4081ce1bcab1da9bbdf61a76